include/mppic/motion_model.hpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 FastSense, Samsung Research
+#pragma once
+
+#include "mppic/models/state.hpp"
+#include <xtensor/xmath.hpp>
+#include <xtensor/xmasked_view.hpp>
+
+#include "mppic/parameters_handler.hpp"
+
+namespace mppi
+{
+
+enum class MotionModelType
+{
+  DiffDrive,
+  Omni,
+  Ackermann
+};
+
+
+inline bool isHolonomic(MotionModelType type)
+{
+  switch (type) {
+    case MotionModelType::Omni:
+      return true;
+    case MotionModelType::Ackermann:
```

Comment:
Could just add a return false here for code cleanliness

Reply:
this was removed